src/psyclone/psyir/symbols/containersymbol.py

```diff
@@ -48,17 +48,10 @@ class ContainerSymbol(Symbol):
     when needed.
 
     :param str name: name of the symbol.
-
-    :raises TypeError: if the name is not a string
     '''
     def __init__(self, name):
         super(ContainerSymbol, self).__init__(name)
 
-        if not isinstance(name, str):
```

Comment:
Out of interest. Why did you remove this?

Reply:
The name attribute must be str in all the kind of symbols. So this check should be inside `super().__init__()`. Note that I did not remove the associated unit-test, so it still works exactly the same.